src/lib/state.ts

```diff
@@ -36,17 +36,23 @@ function getNodeValues(node: ZWaveNode): ValueState[] {
   );
 }
 
-export const dumpValue = (
-  node: ZWaveNode,
-  valueId: TranslatedValueID
-): ValueState => {
-  const valueState = valueId as ValueState;
-  valueState.metadata = node.getValueMetadata(valueId);
-  valueState.value = node.getValue(valueId);
+export const dumpValue = (node: ZWaveNode, valueArgs: any): ValueState => {
+  const valueState = valueArgs as ValueState;
```

Comment:
why is this necessary and can't we put that in the function signature?

Reply:
I want to make sure that all value related messages generate the same valueState message.
So, the value can come from an event but also from enumerating the valueID's on the node.
It's either duplicating code or have one place that contains this logic.

Reply:
FYI: we add the ccVersion and metadata to all value messages.

Reply:
But if you want I could also just do the specifics in each function and use the dumpValue only for the dumpState

Reply:
I'm just scared that we accidentally break something and typing won't find it. Is this line still necessary even with the translated value id type?

Reply:
Yes because we “convert” the incoming ValueID to ValueState, which includes value, metadata and ccVersion.

this line was already here btw, before this change.